ckanext/developerpage/helpers.py

```
@@ -43,11 +44,34 @@ def load_average_5min():
             'load_average_5min' : '{:.2f}'.format(psutil.getloadavg()[1]),
         }
 
+def creation_date(path_to_file):
+    """
+    Try to get the date that a file was created, falling back to when it was
+    last modified if that isn't possible.
+    See http://stackoverflow.com/a/39501288/1709587 for explanation.
+    """
+    if platform.system() == 'Windows':
+        return os.path.getctime(path_to_file)
+    else:
+        stat = os.stat(path_to_file)
+        try:
+            return stat.st_birthtime
+        except AttributeError:
+            # We're probably on Linux. No easy way to get creation dates here,
+            # so we'll settle for when its content was last modified.
+            return stat.st_mtime
+
 def get_host_info():
     python_platform = {
-            'machine_architecture' : platform.machine(),
-            'python_version' : platform.python_version(),
-        }
+            u'machine_architecture': platform.machine(),
+            u'python_version': platform.python_version(),
+            u'host_time_zone': os.environ["TZ"],
+    }
+    try:
+        python_platform[u'config_creation_date'] = datetime.utcfromtimestamp(creation_date('/srv/app/production.ini')).strftime(
+                '%Y-%m-%d %H:%M:%S %z')
+    except Exception as error:
```

@bohdanbobrowski can you please fix this to test for, and report the actual error? (rather than just a catch-all exception)

---

I've removed this try-catch, i feel it's not needed anymore.